sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -181,6 +178,29 @@ def test_spectral_embedding_amg_solver(seed=36):
     assert _check_with_col_sign_flipping(embed_amg, embed_arpack, 0.05)
 
 
+def test_spectral_embedding_amg_solver_failure(seed=36):
+    # Test spectral embedding with amg solver failure, see issue #13393
+    pytest.importorskip('pyamg')
+
+    # The generated graph below is NOT fully connected if n_neighbors=3
+    n_samples = 200
+    n_clusters = 3
+    n_features = 3
+    centers = np.eye(n_clusters, n_features)
+    S, true_labels = make_blobs(n_samples=n_samples, centers=centers,
+                                cluster_std=1., random_state=42)
+
```

Comment:
May be check separately norm_laplacian = False and norm_laplacian = True . The latter is the default, the only option currently checked.